parcels/fieldset.py

```diff
@@ -239,8 +239,15 @@ def from_nemo(cls, filenames, variables, dimensions, indices=None, mesh='spheric
 
         dimension_filename = filenames.pop('mesh_mask')
 
+        interp_method = {}
+        for v in variables:
+            if v in ['U', 'V', 'W']:
+                interp_method[v] = 'cgrid_linear'
+            else:
+                interp_method[v] = 'linear'
```

Comment:
Should we let freedom to the user?
I am thinking if he wants to interpolate the tracer with `'nearest'`
(This is a valid choice. I guess tracer are considered constant per cell when the integral over the domain is computed, but I'm not sure)

Watch out, a tracer field does not live on the same grid as U and V
=> line 223 should be modified giving `dimensions['U'], dimensions['Salt']` (in the doc)